components/common/chip/chip.tsx

```diff
@@ -10,10 +10,11 @@ import RenderEmoji from 'components/emoji/render_emoji';
 // This component is a temporary placeholder for use until the authoritative `compass-components` Chip is implemented.
 
 type Props = {
-    onClick: () => void;
-    id: string;
-    defaultMessage: string;
-    values: Record<string, any>;
+    onClick?: () => void;
+    id?: string;
+    defaultMessage?: string;
+    values?: Record<string, any>;
```

Comment:
not part of your changes, but is the `any` really our best option here?

Reply:
maybe use a generic for it???. At the end this contains the possible replaceable values for the FormattedMessage component.

Reply:
fair enough!